src/pybind/mgr/dashboard/frontend/src/app/core/auth/login/login.component.scss

```diff
@@ -1,31 +1,21 @@
-@import '../../../../defaults';
-
-.login {
-  height: 100%;
-
-  .row {
-    color: $color-login-row-text;
-    background-color: $color-login-row-bg;
-  }
-
-  h1 {
-    margin-top: 0;
-    margin-bottom: 30px;
-  }
+.branding-border {
+  width: calc(100% + 60px);
+  margin-left: -30px;
+  height: 3px;
+  background-color: #c00;
+}
 
-  .btn-password,
-  .form-control {
-    color: $color-password-toggle-text;
-    background-color: $color-password-toggle-bg;
-  }
 
-  .btn-password:focus {
-    outline-color: $color-password-toggle-focus;
-  }
+.login-logo > img{
+  display: block;
+  width: 125px;
+  margin: 30px auto;
+}
 
-  .checkbox-primary input[type='checkbox']:checked + label::before,
-  .checkbox-primary input[type='radio']:checked + label::before {
-    background-color: $color-login-checkbox-bg;
-    border-color: $color-login-checkbox-border;
-  }
+.login-pf-page-footer{
+  width:30%;
+  position:fixed;
```

Comment:
Maybe this is the setting that we should tune to put the footer together with the main form?

Reply:
@epuertat It is based on the branding [presentation](https://docs.google.com/presentation/d/1uLTmaNtdn0hgWSk5N9i5c4amXlpLqMVsOsHTfxx02wY/edit?usp=sharing) prepared by @julienlim